ansible_collections/arista/avd/plugins/plugin_utils/eos_designs_facts/uplinks.py

```diff
@@ -182,6 +182,9 @@ def uplinks(self: EosDesignsFacts) -> list:
                 if self.shared_utils.uplink_interface_speed is not None:
                     uplink["speed"] = self.shared_utils.uplink_interface_speed
 
+                if self.shared_utils.uplink_switch_interface_speed is not None:
+                    uplink["peer_speed"] = self.shared_utils.uplink_switch_interface_speed
+
```

Comment:
These lines need to be added under the `p2p` uplink_type as well.

Reply:
Gah, stupid mistake.  Should be fixed in latest commit.